### PR TITLE
DynamoDB Scala Connectivity Sample EMR Studio Notebook

### PR DESCRIPTION
**Pull request by vivekgautam104**
*Issue #, if available:*

*Description of changes:* DynamoDB Scala Connectivity Sample EMR Studio Notebook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/dynamodb-with-spark.ipynb [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    "     \n",
+    "     a. Open the Livy configuration file\n",
+    "     \n",
+    "     `vim /etc/livy/conf/livy.conf`\n",
+    "     \n",
+    "     b. Add the ddb jars:\n",
+    "     \n",
+    "     `livy.file.local-dir-whitelist   /usr/share/aws/emr/ddb/lib/`\n",
+    "     \n",
+    "     c. Restart Levi\n",
```
Comment (emrnotebooks) [inline examples/dynamodb-with-spark.ipynb:57]: Typo: Livy

File: examples/dynamodb-with-spark.ipynb [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+   "cell_type": "markdown",
+   "id": "7329c0d3",
+   "metadata": {},
+   "source": [
+    " 1. EMR Version - emr-6.4.0\n",
+    " 2. The notebook is using Amazon EMR-DynamoDB Connector(emr-ddb-hadoop.jar) which is used to access data in Amazon DynamoDB using Apache Hadoop, Apache Hive, and Apache Spark in Amazon EMR. (https://docs.aws.amazon.com/emr/latest/ReleaseGuide/EMRforDynamoDB.html)\n",
+    " 3. The Amazon EMR-DynamoDB Connector is locally available within EMR at location below: \n",
+    " \n",
+    " `/usr/share/aws/emr/ddb/lib/emr-ddb-hadoop.jar`\n",
+    " 4. To locally use the spark.jars in the configuration using the `file://` protocol, Livy does not allow local files to be uploaded to the user session by default. So we need to update the livy configuration to allow the local directory and restart livy server to have these take effects.\n",
```
Comment (emrnotebooks) [inline examples/dynamodb-with-spark.ipynb:47]: Please explicitly mention that below commands should be run on the EMR cluster.

Comment (vivekgautam104) [reply examples/dynamodb-with-spark.ipynb:47]: Added a statement:
" ##### The following commands should be run from the EMR cluster attached to the notebook\n",

File: examples/dynamodb-with-spark.ipynb (hunk 1 above)
Comment (vivekgautam104) [reply examples/dynamodb-with-spark.ipynb]: Fixed

**Pull request merged**